modules/Layout/CookieConsentBar/CookieConsentBar.tsx

```diff
@@ -0,0 +1,56 @@
+import { CookieConsentBar as DefaultCookieConsentBar } from '@prezly/analytics-nextjs';
+import { useCompanyInformation } from '@prezly/theme-kit-nextjs';
+import translations from '@prezly/themes-intl-messages';
+import { FormattedMessage } from 'react-intl';
+
+import Button from '../../../components/Button';
+
+function CookieConsentBar() {
+    const { cookie_statement: cookieStatement } = useCompanyInformation();
+
+    return (
+        <DefaultCookieConsentBar>
+            {({ onAccept, onReject }) => (
+                <div className="border-b border-primary-darkest py-12 mb-10">
+                    <div className="container">
+                        <div className="grid grid-cols-1 gap-y-4 lg:grid-cols-2 lg:gap-x-4">
+                            <div className="flex flex-col">
+                                <p className="font-bold text-2xl mb-2">
+                                    <FormattedMessage {...translations.cookieConsent.title} />
+                                </p>
+                                {cookieStatement ? (
+                                    <div dangerouslySetInnerHTML={{ __html: cookieStatement }} />
+                                ) : (
+                                    <p>
+                                        <FormattedMessage
+                                            {...translations.cookieConsent.description}
+                                        />
+                                    </p>
+                                )}
+                            </div>
+                            <div className="flex lg:justify-center lg:items-end flex-col">
+                                <div className="flex md:flex-row flex-col gap-y-4 md:gap-y-0 md:gap-x-4 mb-4">
+                                    <button
+                                        type="button"
+                                        className="border py-3 px-4 rounded-lg border-primary-dark"
+                                        onClick={onReject}
+                                    >
```

Comment:
This buttons is missing any interactions styling 😕 
Also, wouldn't it be better to extend the current `Button` component to add a `secondary` variant?

Reply:
I guess it makes more sense to add an `outlined` variant, let me come up with an implementation for that with some interaction styling and we can ask @alpower if that looks good.